storybook/pages/CommunitiesPortalLayoutPage.qml

```diff
@@ -51,8 +51,8 @@ SplitView {
                     }
                 }
 
-                function navigateToCommunity() {
-                    logs.logEvent("CommunitiesStore::navigateToCommunity", ["communityId"], arguments)
```

Comment:
unfortunately it's not correct as it will print e.g.:
```
11:20:43: CommunitiesStore::navigateToCommunity: [communityId: "i", "d", "1"]
```
similarly is some other places

Reply:
Hmm... I'll have a look